modules/miscutil/lib/mailutils.py

```diff
@@ -24,24 +24,24 @@
 
 __revision__ = "$Id$"
 
+import os
+import re
 import sys
+from cStringIO import StringIO
 from time import sleep
+
+import six
```

Comment:
If `six` is required, you may need to amend the `INSTALL` file and the `requirements.txt` file about this.